zeeguu/core/test/test_retrieve_and_compute.py

```diff
@@ -22,7 +26,9 @@ def setUp(self):
 
     def testDifficultyOfFeedItems(self):
         feed = FeedRule().feed1
-        download_from_feed(feed, zeeguu.core.model.db.session, 3, False)
+        crawl_report = CrawlReport()
+        crawl_report.add_feed(feed.language.code, feed.id)
```

Comment:
a `Feed` object knows it's `language.code` and `id` so in principle, you could only expect a `Feed` object in the `add_feed` method?

Reply:
That's a good point, I will make that change!